frontend/packages/knative-plugin/src/catalog/event-source-data.ts

```diff
@@ -42,4 +43,10 @@ export const getEventSourceCatalogProviderData = (
       ),
       provider: 'Red Hat',
     },
+    [EventSourceCephModel.kind]: {
+      description: t(
+        'knative-plugin~Used to connect OpenShift managed applications to Ceph notifications.',
+      ),
+      provider: 'Red Hat',
```

Comment:
also, we may need to check with the CEPH team as this has not contributed to the serverless operator provided by Red Hat.